common/src/main/java/com/wynntils/features/combat/RangeVisualizerFeature.java

```diff
@@ -120,21 +122,36 @@ private void checkMajorIdCircles(Player player) {
         // Offset the radius slightly so multiple circles can be shown for each player
         // Only a few major IDs can actually be applied at the same time, but we make this general
         List<Pair<CustomColor, Float>> circles = validGear.stream()
-                .flatMap(gearInfo -> gearInfo.fixedStats().majorIds().stream().map(majorId -> switch (majorId.name()) {
-                    case "Taunt" -> Pair.of(CommonColors.ORANGE.withAlpha(TRANSPARENCY), 12f);
-                    case "Saviour’s Sacrifice" -> Pair.of(CommonColors.WHITE.withAlpha(TRANSPARENCY), 8f);
-                    case "Heart of the Pack" -> Pair.of(CommonColors.PINK.withAlpha(TRANSPARENCY), 8.1f);
-                    case "Guardian" -> Pair.of(CommonColors.RED.withAlpha(TRANSPARENCY), 7.9f);
-                    default -> null;
-                }))
+                .flatMap(gearInfo ->
+                        gearInfo.fixedStats().majorIds().stream().map(majorId -> circleFromMajorId(majorId.name())))
                 .filter(Objects::nonNull)
-                .toList();
+                .collect(Collectors.toList());
+
+        // add circles gained from raid major id buffs
```

Comment:
Perhaps keep this a stream, make the code below also be a stream, concat them and then put them into a list. That'd keep the code style the same, and also shorten the code a tiny bit.